molgenis-app/pom.xml

```diff
@@ -40,25 +38,38 @@
         Runs by default in the deploy phase,
         if you need more tags you can run the goals manually -->
       <plugin>
-        <groupId>com.spotify</groupId>
-        <artifactId>dockerfile-maven-plugin</artifactId>
-        <version>${dockerfile-maven-plugin.version}</version>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>${jib-maven-plugin.version}</version>
         <executions>
           <execution>
-            <id>default</id>
+            <phase>deploy</phase>
             <goals>
               <goal>build</goal>
-              <goal>push</goal>
             </goals>
           </execution>
         </executions>
         <configuration>
-          <repository>${dockerfile.repository}</repository>
-          <tag>${dockerfile.tag}</tag>
-          <buildArgs>
-            <WAR_FILE>${dockerfile.warfile}</WAR_FILE>
-          </buildArgs>
-          <useMavenSettingsForAuth>true</useMavenSettingsForAuth>
+          <from>
+            <image>registry://registry.molgenis.org/molgenis/molgenis-base:PR-74</image>
```

Comment:
Is the base image introduced in https://github.com/molgenis/molgenis-ops-docker/pull/74